jaxrs-api/src/main/java/jakarta/ws/rs/core/Response.java

```diff
@@ -1350,6 +1350,13 @@ public enum Status implements StatusType {
          * @since 2.1
          */
         PRECONDITION_REQUIRED(428, "Precondition Required"),
+        /**
+         * 422 Unprocessable Entity, see <a href="https://httpwg.org/specs/rfc9110.html#status.422">RFC 9110: 
+         * 422 Unprocessable Content</a>.
+         *
+         * @since 4.0
```

Comment:
4.0 is already released. We should have the next version here.

Reply:
@mkarg What version should I use?

Reply:
@spericas Forwarding question to project lead to trigger a final decision if we want a 4.1 or directly work towards 5.0.